### PR TITLE
clarify next equivalent round

### PR DESCRIPTION
**Pull request by christ1ne**
@tjablin @petermattson @DilipSequeira

### CODE REVIEW

**Comment by tjablin** [comment]
I don't have permission to assign myself as a reviewer, but LGTM. Maybe clarify what an "equivalent" submission is? Is Mobile Inference the same as Datacenter Inference?

**Comment by guschmue** [comment]
you have the powers now

**Comment by johntran-nv** [comment]
@christ1ne what does "equivalent" mean?

**Comment by christ1ne** [comment]
> @christ1ne what does "equivalent" mean?

training -> next training round
inference -> next inference round

maybe we can rephrase it to "next submission round of the same type" if that helps!

**Review by tjablin — APPROVED** [review]
